packages/app/src/runner/index.ts

```diff
@@ -283,17 +309,30 @@ function runSpecE2E (spec: SpecFile) {
 
   const $autIframe: JQuery<HTMLIFrameElement> = autIframe.create().appendTo($container)
 
+  // Remove the spec bridge iframe
+  document.querySelectorAll('iframe.spec-bridge-iframe').forEach((el) => {
+    el.remove()
+  })
+
   autIframe.showInitialBlankContentsE2E()
 
   // create Spec IFrame
   const specSrc = getSpecUrl(config.namespace, encodeURIComponent(spec.relative))
 
-  const $specIframe = createSpecIFrame(specSrc)
+  // FIXME: BILL Determine where to call client with to force browser repaint
+  /**
+   * call the clientWidth to force the browser to repaint for viewport changes
```

Comment:
Need to find the appropriate place to put this line of code where applicable. original reference [here](https://github.com/cypress-io/cypress/pull/18075/files#diff-bd8f567e7cefe6212c99b7760b613f12ffa518138f1d19aff5f3e5cf0c677bf9R234)